src/nlr/NetworkLevelReasoner.h

```diff
@@ -136,6 +136,8 @@ class NetworkLevelReasoner : public LayerOwner
     List<PiecewiseLinearConstraint *> getConstraintsInTopologicalOrder();
     void addConstraintInTopologicalOrder( PiecewiseLinearConstraint *constraint );
 
+    void removeConstraintFromTopologicalOrder( PiecewiseLinearConstraint *constraint );
```

Comment:
Is this function used anywhere?

Reply:
Thanks for the catch. I removed this method.

Reply:
This is now used in the preprocessor.